mountpoint-s3/src/fs/config.rs

```diff
@@ -87,6 +87,8 @@ pub struct CacheConfig {
     pub file_ttl: Duration,
     /// How long the kernel will cache metadata for directories
     pub dir_ttl: Duration,
+    /// How long the kernel will cache negative entries
```

Comment:
File and directory TTLs are sent to the kernel, the negative cache is implemented only in Mountpoint.
```suggestion
    /// How long the file system will cache negative entries
```